config/locales/en-US.yml

```diff
@@ -3,6 +3,10 @@
 # Use this as the base for the locale file of your language.
 
 "en-US":
+  admin:
+    blacklist:
+      emails_found: "%{emails_count} emails found"
```

Comment:
Minor wibble if we are doing this it would be ( I think ) from ( http://guides.rubyonrails.org/i18n.html#pluralization ) also see https://github.com/otwcode/otwarchive/pull/2281

blacklist:
  emails_found:
     one: an email found
     other: "%{emails_count} emails found"